hhs_hosp/delphi_hhs/constants.py

```diff
@@ -7,3 +7,9 @@
     CONFIRMED,
     SUM_CONF_SUSP
 ]
+
+# incoming geo: state
+GEOS = [
```

Comment:
Since some of the other indicators use the geos variable to indicate all geos, not just additional ones, would it be more clear to rename this as ADDITIONAL_GEOS? Alternatively, could add state and deal with it in the code, or just write a longer comment about this

Reply:
that's fair, will convert to all geos and handle state conditionally in code